src/rpcnet.cpp

```diff
@@ -165,6 +173,18 @@ UniValue getpeerinfo(const UniValue& params, bool fHelp)
         }
         obj.push_back(Pair("whitelisted", stats.fWhitelisted));
 
+        UniValue sendPerCmd(UniValue::VOBJ);
+        BOOST_FOREACH( const mapCmdSize::value_type &i, stats.mapSendBytesPerCmd) {
+            sendPerCmd.push_back(Pair(i.first, i.second));
+        }
+        obj.push_back(Pair("bytessent_per_cmd", sendPerCmd));
+
+        UniValue recvPerCmd(UniValue::VOBJ);
+        BOOST_FOREACH( const mapCmdSize::value_type &i, stats.mapRecvBytesPerCmd) {
+            recvPerCmd.push_back(Pair(i.first, i.second));
```

Comment:
OK i agree. If you want to know eg. the set of commands that is not considered *other*. OK I don't care deeply about this, fine to leave it as-is, but I expect more people to comment on this.